src/mprk.jl

```diff
@@ -280,6 +266,75 @@ function perform_step!(integrator, cache::MPEConstantCache, repeat_step = false)
     integrator.u = u
 end
 
+function perform_step_nonconservative!(integrator, cache::MPEConstantCache,
```

Comment:
The functions `perform_step_nonconservative`!  and `perform_step_conservative`! only differ in three lines. 
- Line 279: Evaluation of `f.d`
- Line 286: Adding `dt * diag(P)` to the linear system's right hand side.
- Line 287: Passind `d` to `build_mprk_matrix`.

@ranocha  Do we really need these two versions of `perform_step` or should we only have one and use `if` to handle both `ConservativePDSFunction` and `PDSFunction`?

Reply:
I think you're right - it will be easier to maintain the code if we keep it together.